.github/aggregate.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs')
+const process = require('process')
+const path = require('path')
+
+// params: list of json reports to aggregate
+const inputs = process.argv.slice(2)
+
+const processReport = (filePath) => {
+  const resolvedPath = path.isAbsolute(filePath) ? filePath : path.resolve(process.cwd(), filePath);
```

Comment:
I'll drop the option, but what do you mean by chaos? What's the risk?

Reply:
That's a tautology; what problem caused by resolving relative or absolute path are we trying to prevent here?

I'm fine dropping it, but I'd like to know if there is something I should be aware of when it comes to nodejs scripts and pathing. I'm mostly using tool that allows both paths, it makes it easier to work and develop with (`cat /etc/hosts ./README.md`, or `node ./myscript`, `node ~/myscript`).